web/src/settings_config.ts

```diff
@@ -789,7 +789,7 @@ export const all_notification_settings = [
     ...stream_notification_settings,
 ];
 
-type Settings = UserSettings | RealmDefaultSettings;
+export type Settings = UserSettings | RealmDefaultSettings;
```

Comment:
Can we give this a more globally meaningful name for exporting, like `GenericUserSettings`

Reply:
By "for exporting" did you mean exporting this with a different name( `GenericUserSettings` as you suggested) . Something like 
```
 type Settings = UserSettings | RealmDefaultSettings;
export type {Settings as GenericUserSettings};
```
or renaming the type altogether.

```
export type GenericUserSettings = UserSettings | RealmDefaultSettings
```